bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/GuideGroupSpec.scala

```diff
@@ -18,11 +16,12 @@ import scala.collection.JavaConverters._
 import scalaz._, Scalaz._
 
 class GuideGroupSpec extends Specification with ScalaCheck with Arbitraries {
+
   import GuideGroupSpec.AllProbes
 
   "GuideGroup name" should {
     "always be defined for manual groups, undefined for automatic" in
-      forAll { (g: GuideGroup) =>
+      forAll { g: GuideGroup =>
```

Comment:
If you look at `TargetSpec`, `AngleSpec` etc., where we have a type tag like this we surround them in parens.  We often need two or more arbitrary things in which case parens become necessary.  I think I'd rather it all be consistent and just use parens when there are type tags.